fs/dafsclient/rmtclient.cpp

```diff
@@ -112,6 +112,12 @@ static class _securitySettingsClient
         }
     }
 
+    const IPropertyTree * getSecureConfig()
+    {
+        //Later: return a synced tree...
```

Comment:
When? Is that for a different PR? Is there a Jira open to revisit if so?

Reply:
HPCC-30753